.github/workflows/cdktf-provider-docs-rollout.yml

```diff
@@ -0,0 +1,61 @@
+name: CDKTF Provider Docs Rollout
+
+on:
+  workflow_dispatch: {}
+  schedule:
+    - cron: 0 0 * * 1
+
+jobs:
+  cdktfDocs:
+    strategy:
+      fail-fast: false
+      matrix:
+        provider:
+          # To onboard new providers, add a new entry to this matrix and make sure the team-tf-cdk github user has write permissions to the repository.
```

Comment:
```suggestion
          # NOTE: This only works for Terraform providers in the HashiCorp Github org
          # To onboard new providers, add a new entry to this matrix and make sure the team-tf-cdk github user has write permissions to the repository.
```

Reply:
If this is true, I'd like to add this note (which will also serve future me 😄)

Reply:
It’s not, if some other Provider repo gives push rights to our users it works for them as well